src/Sinch/SinchClient.cs

```diff
@@ -271,27 +266,22 @@ public ISinchAuth Auth
 
         /// <inheritdoc />
         public ISinchVoiceClient Voice(string appKey, string appSecret,
-            CallingRegion? callingRegion = default,
-            AuthStrategy authStrategy = AuthStrategy.ApplicationSign)
+            CallingRegion? callingRegion = default)
         {
             if (string.IsNullOrEmpty(appKey))
                 throw new ArgumentNullException(nameof(appKey), "The value should be present");
 
             if (string.IsNullOrEmpty(appSecret))
                 throw new ArgumentNullException(nameof(appSecret), "The value should be present");
 
-            ISinchAuth auth;
-            if (authStrategy == AuthStrategy.ApplicationSign)
-                auth = new ApplicationSignedAuth(appKey, appSecret);
-            else
-                auth = new BasicAuth(appKey, appSecret);
+            ISinchAuth auth = new ApplicationSignedAuth(appKey, appSecret);
```

Comment:
Maybe you can declare the type as `ApplicationSignedAuth` directly to avoid the cast later on as you are now supporting only this kind of authentication

Reply:
nah, http use interface

Reply:
I don't understand the link with http here. Can you elaborate? To me, it's related to the `SinchVoiceClient` constructor parameters only. Did I miss something specific to cs?

Reply:
Http class constructed below depends on `ISinchAuth`

Reply:
Thanks for the precision, I was focusing on the modified code